From 1cead3cbb02f7058846a6f7b7a42d875b27ee64a Mon Sep 17 00:00:00 2001
From: Alexandre Derumier <aderumier@odiso.com>
Date: Fri, 15 Jun 2018 00:36:03 +0200
Subject: [PATCH 02/12] add dummy mtu, bridgevlanport modules

As pve-manager package still provide ifupdown custom bridgevlanport && mtu script,
we can override them with ifupdown2 addon scripts, doing nothing.

(ifupdown2 don't launch an ifdown script, if an ifupdown2 addon script
 exist with same name)

Signed-off-by: Thomas Lamprecht <t.lamprecht@proxmox.com>
Signed-off-by: Alexandre Derumier <aderumier@odiso.com>
---
 etc/network/ifupdown2/addons.conf  |  2 ++
 ifupdown2/addons/bridgevlanport.py | 17 +++++++++++++++++
 ifupdown2/addons/mtu.py            | 17 +++++++++++++++++
 3 files changed, 36 insertions(+)
 create mode 100644 ifupdown2/addons/bridgevlanport.py
 create mode 100644 ifupdown2/addons/mtu.py

diff --git a/etc/network/ifupdown2/addons.conf b/etc/network/ifupdown2/addons.conf
index 726d63a..a8f2317 100644
--- a/etc/network/ifupdown2/addons.conf
+++ b/etc/network/ifupdown2/addons.conf
@@ -20,6 +20,8 @@ up,dhcp
 up,address
 up,addressvirtual
 up,usercmds
+up,mtu
+up,bridgevlanport
 post-up,usercmds
 post-up,clagd
 post-up,vxrd
diff --git a/ifupdown2/addons/bridgevlanport.py b/ifupdown2/addons/bridgevlanport.py
new file mode 100644
index 0000000..ab6a680
--- /dev/null
+++ b/ifupdown2/addons/bridgevlanport.py
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+#
+
+import os
+
+try:
+    from ifupdown2.ifupdown.utils import utils
+
+    from ifupdown2.ifupdownaddons.modulebase import moduleBase
+except ImportError:
+    from ifupdown.utils import utils
+
+    from ifupdownaddons.modulebase import moduleBase
+
+class bridgevlanport(moduleBase):
+
+    _modinfo = {'mhelp' : 'dummy package to override proxmox script'}
diff --git a/ifupdown2/addons/mtu.py b/ifupdown2/addons/mtu.py
new file mode 100644
index 0000000..efdf569
--- /dev/null
+++ b/ifupdown2/addons/mtu.py
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+#
+
+import os
+
+try:
+    from ifupdown2.ifupdown.utils import utils
+
+    from ifupdown2.ifupdownaddons.modulebase import moduleBase
+except ImportError:
+    from ifupdown.utils import utils
+
+    from ifupdownaddons.modulebase import moduleBase
+
+class mtu(moduleBase):
+
+    _modinfo = {'mhelp' : 'dummy package to override proxmox script'}
-- 
2.30.2

